plugins/auth-node/src/oauth/createOAuthRouteHandlers.ts

```diff
@@ -320,7 +320,9 @@ export function createOAuthRouteHandlers<TProfile>(
           providerInfo: {
             idToken: result.session.idToken,
             accessToken: result.session.accessToken,
-            scope: result.session.scope,
```

Comment:
Hi, We are seeing a similar issue where this issue does not happen during first login to github using authenticator app, but happens during subsequent login after logging out of github and logging in again.

This issue can be reproduced with below steps.

Go to page where Github authenticator sign in present
Login to github using Github authenticator
Go to page where 'backstage-plugin-github-pull-requests' results should be displayed. The results are displayed
Go to page where Github authenticator sign in present
Log out of gihub
login to github again
Go to page where 'backstage-plugin-github-pull-requests' results should be displayed. The results are not displayed and asks for github login again.
I see that the fix for "scope: result.session.scope," was made only to one location in file plugins/auth-node/src/oauth/createOAuthRouteHandlers.ts but it is present at two locations. Can this be an issue?